web_widget_image_download/static/src/css/web_widget_image_download.css

```diff
@@ -0,0 +1,6 @@
+/* Copyright 2016 Flavio Corpa <flavio.corpa@tecnativa.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+.oe_form_binary_file_download {
+  color: #eee !important;
```

Comment:
@Yajo you are right, I didn't want to include it, but there was another class from odoo that override my widget custom css, so I had to do it 😕